projects/element-translate-ng/translate/si-localize.ts

```diff
@@ -32,3 +32,11 @@ export const initSiLocalize = (): void => {
   (Zone.current as any)._properties.siResolveLocalize = siResolveLocalize;
   globalScope.$localize = $localize;
 };
+
+// Init $localize in the current zone
```

Comment:
WDYT if we're a bit more specific here?

```suggestion
// Init $localize in the current zone if not already available
```

Reply:
Slighlty had to adjust to calls, but comment is improved as well.